packaging/resource_suite_s3_nocache.py

```diff
@@ -2094,3 +2094,460 @@ def test_missing_keyfile(self):
 
             # cleanup
             self.admin.assert_icommand("iadmin rmresc s3resc1", 'EMPTY')
+
+
+class Test_S3_NoCache_Glacier_Base(session.make_sessions_mixin([('otherrods', 'rods')], [('alice', 'apass'), ('bobby', 'bpass')])):
+
+    def __init__(self, *args, **kwargs):
+        """Set up the cacheless test."""
+        # if self.proto is defined use it else default to HTTPS
+        if not hasattr(self, 'proto'):
+            self.proto = 'HTTPS'
+
+        # if self.archive_naming_policy is defined use it
+        # else default to 'consistent'
+        if not hasattr(self, 'archive_naming_policy'):
+            self.archive_naming_policy = 'consistent'
+
+        super(Test_S3_NoCache_Glacier_Base, self).__init__(*args, **kwargs)
+
+    def setUp(self):
+
+        super(Test_S3_NoCache_Glacier_Base, self).setUp()
+        self.admin = self.admin_sessions[0]
+        self.user0 = self.user_sessions[0]
+        self.user1 = self.user_sessions[1]
+
+        print("run_resource_setup - BEGIN")
+        self.testfile = "testfile.txt"
+        self.testdir = "testdir"
+
+        hostname = lib.get_hostname()
+
+        # read aws keys
+        self.read_aws_keys()
+
+        # set up s3 bucket
+        if self.proto == 'HTTPS':
+            s3_client = Minio(self.s3endPoint, access_key=self.aws_access_key_id, secret_key=self.aws_secret_access_key, region = self.s3region)
+        else:
+            s3_client = Minio(self.s3endPoint, access_key=self.aws_access_key_id, secret_key=self.aws_secret_access_key, region = self.s3region, secure=False)
```

Comment:
Can be a one liner. See earlier comment.

Reply:
Done for all.